src/prng/prng.lisp

```diff
@@ -27,27 +20,42 @@
 
 (defmethod make-prng :around (name &key (seed :random))
```

Comment:
I really, _really_ wanted to delete `MAKE-PRNG`, but it was an exported part of Ironclad's interface.

It might actually be a good idea to delete it, to force people not to allocate their own Fortuna generators but instead just use the OS one.